src/types/Project.tsx

```diff
@@ -1,6 +1,8 @@
 export type Project = {
-  id: number,
-  author: string,
-  title: string,
-  create_date: Date,
-}
\ No newline at end of file
+  id: number;
+  author: string;
+  title: string;
+  create_date: Date;
+  intro: string;
+  tech: string[];

```

Comment:
maybe tech**s** could be a better name? or something else

Reply:
renamed to techStacks :)